miso-web/src/main/webapp/pages/editLibrary.jsp

```diff
@@ -240,7 +240,7 @@
   </tr>
 </c:if>
 <c:choose>
-  <c:when test="${!empty library.sample && !empty library.sample.sampleAdditionalInfo && !empty library.libraryAdditionalInfo}">
+  <c:when test="${!empty library.sample && !empty library.sample && !empty library.libraryAdditionalInfo}">
```

Comment:
Checking same condition twice. Looks like this was originally intended to check if it was a detailed sample